server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/DataArchiveTask.java

```diff
@@ -84,6 +84,7 @@ protected boolean doStart(JobContext context) throws Exception {
             jobStore.setDlmTableUnits(result);
         } catch (Exception e) {
             log.warn("Get dlm job failed!", e);
+            taskContext.getExceptionListener().onException(e);
```

Comment:
how about remove try... block, throw exception straightly,
or, it is hard for Task Implemation to follow task framework's exception handling flow.

Reply:
It's  task inner implement.  base class not know how sub class will handle exception.
Actually, this change should not involved in this PR, cause how exception thrown and notify is the duty of task code owner.